integration-cli/docker_cli_service_logs_test.go

```diff
@@ -277,7 +283,7 @@ func (s *DockerSwarmSuite) TestServiceLogsTTY(c *check.C) {
 	// and make sure we have all the log lines
 	waitAndAssert(c, defaultReconciliationTimeout, countLogLines(d, name), checker.Equals, 2)
 
-	cmd := d.Command("service", "logs", name)
+	cmd := d.Command("service", "logs", "--pretty=false", name)
```

Comment:
Yes. TTY logs require `--pretty=false` because we cannot reliably parse the details of TTY logs and therefore cannot pretty-print them. This is the only CLI usage change for the defaults.